src/string_bytes.cc

```diff
@@ -857,7 +855,16 @@ Local<Value> StringBytes::Encode(Isolate* isolate,
                                  const uint16_t* buf,
                                  size_t buflen) {
   Local<String> val;
-
+  uint16_t* dst = NULL;
```

Comment:
Seeing how you're writing new code here, you might as well replace this with `std::vector<uint16_t>` so there is no need to explicitly free the memory.